updater/lib/dependabot/updater/operations/refresh_version_pull_request.rb

```diff
@@ -0,0 +1,346 @@
+# frozen_string_literal: true
+
+# This class implements our strategy for 'refreshing' an existing Pull Request
+# that updates a dependnency to the latest permitted version.
+#
+# It will determine if the existing diff is still relevant, in which case it
+# functions similar to a "rebase", but in the case where the project folder's
+# dependencies have changed or a newer version is available, it will supersede
+# the existing pull request with a new one for clarity.
+module Dependabot
+  class Updater
+    module Operations
+      class RefreshVersionPullRequest
```

Comment:
🤔 While `Update` is inherent in this name, I wonder if it should be included to make a stronger distinction between it and a security update. I haven't spent time with it, so this is more of a knee-jerk reaction.

Since you spent time with this, what are your thoughts on `RefreshVersionUpdatePullRequest` vs `RefreshVersionPullRequest`?

Reply:
Yeah, I think I've started biasing towards avoiding the word `Update` as it feels really implicit, but I also think the 'nouns' are a little week in terms of a PR that updates to the latest version vs a PR that updates to a secure version, etc.

We have very fixed languages/ideas in the service and I'm trying not to bleed them into Core/the Updater, but it probably makes sense to just use `VersionUpdate` `SecurityUpdate` to be honest?

Reply:
> We have very fixed languages/ideas in the service and I'm trying not to bleed them into Core/the Updater, but it probably makes sense to just use VersionUpdate SecurityUpdate to be honest?

I think it does make sense. I think security updates are a semi-abstract idea in Core, so naming them might make that more concrete. In all of this, I trust your judgment and defering until later is 👍 by me 🙂

Reply:
I went ahead and renamed it as SecurityUpdate/VersionUpdate are the features we talk about in our documentation and for better or worse, the abstraction is through into the Updater.

I think by splitting this up we can start to gain nuance in future, but for now, one naming convention is a little easier to parse.